pages/common/yadm.md

```diff
@@ -0,0 +1,35 @@
+# yadm
+
+> A dotfiles manager that works by using `git`.
+> The subcommands are just like `git` but use `yadm`.
+> Some subcommands such as `init`, `clone`, `push`, and `pull` have their own usage documentation.
+> This command is one that allows you to pass flags to configure yadm.
+> More information: <https://yadm.io/docs/overview>.
```

Comment:
We try to not use more than 3 lines of the description, we sometimes use 4 lines when it adds valuable information. There is no other page with 5 lines of description.

```suggestion
> A dotfiles manager that works by using `git`.
> Some subcommands such as `init`, `clone`, `push`, and `pull` have their own usage documentation.
> More information: <https://yadm.io/docs/overview>.
```